### PR TITLE
[FIX] web_editor: collaboration should be opt-in 

### PR DESCRIPTION
**Pull request by sebgeelen**
The new collaboration feature of the web editor was active by default.
We change this to make it an opt-in option on the field html.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/static/src/js/backend/field_html.js [hunk 1]
```diff
@@ -185,18 +185,21 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
                 res_id: this.res_id,
             },
             placeholder: this.attrs && this.attrs.placeholder,
-            collaborationChannel: {
-                collaborationModelName: this.model,
-                collaborationFieldName: this.name,
-                collaborationResId: parseInt(this.res_id),
-            },
+            collaborationChannel:
+                'collaboration' in this.nodeOptions && this.nodeOptions.collaboration
```
Comment (dmo-odoo) [inline addons/web_editor/static/src/js/backend/field_html.js:189]: Wouldn't this work ?
```
!!this.nodeOptions.collaboration && {
    collaborationModelName: this.model,
    collaborationFieldName: this.name,
    collaborationResId: parseInt(this.res_id),
}
```

Comment (sebgeelen) [reply addons/web_editor/static/src/js/backend/field_html.js:189]: sure

**Comment by dmo-odoo** [comment]
@robodoo r+

**Comment by dmo-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**